### ISSUE TITLE
Review All Button

### ISSUE THREAD

**Issue by jchang358** [SYtor/Kanji-Dojo#49]
Hello,

I'm really liking the app and have been using it for a while now. That being said, I was wondering if we could get some sort of way to review all the kanji marked for review automatically. 

See, I'm pairing this app with Wanikani which has 60 levels. The app already gives me the ability to create this groups. Great! however, I have close to 20 levels now and the home screen doesn't tell me which level need reviews just when it was last reviewed. I have to click on each to check. Moreover, getting inside of a group shows you the actual kanji in the group that needs review (assuming you filter by it) meaning you actually get to see the kanji you need to review before you review it like it or not. This is a big spoiler and gives you the answer before the question. 

so in short it takes too much time to click on each group to check reviews and the reviews are spoiled by characters showing before you reviewing them.

Having just one button above the plus in the home screen to do my daily reviews would be immensely helpful particularly once i get to having 30+ groups of kanjis if i catch up with my wanikani.

Anyways, just my two cents. I appreciate the hard work.

**Comment by syt0r** [comment]
Hm, I'm not sure how usable are reviews of this huge size, but right now you can just create a custom practice with all wanikani characters in it and review it. You can copy characters from here:

https://github.com/syt0r/Kanji-Dojo/blob/develop/parser/src/main/java/ua/syt0r/kanji/parser/WanikaniLevelsKanji.kt

**Comment by jchang358** [comment]
Thanks for the response. How about showing how many reviews are left inside a group and a way to do all those reviews without getting inside and getting the characters spoiled?

I know I can make one group with all wanikani characters in it and do custom review but that means I lose the convenience of easily adding the levels as I progress my wanikani. it also means I need to heavily modify my already existing library. Maybe a "merge groups" would help?

**Comment by syt0r** [comment]
Yeah, merge groups feature might help. But anyway, Kanji Dojo is tracking reviews for specific characters regardless of which practice set the character is in so that the same character might be in different practice sets but it would still have the same review date. So in your situation, you can just create a practice set with wanikani characters you already know and then gradually edit this practice set by gradually adding new levels there whenever you're ready. 
And by the way, new version v1.5 that is currently being rolled out has a menu on the screen with a list of practices that you can use to start review without going into practice details, so there is less spoiling

**Comment by syt0r** [comment]
I guess option to merge practice sets fixed this
